build/build.ps1

```diff
@@ -8,14 +8,44 @@ $all_ok = $True
 
 if ($Env:ENABLE_NATIVE -ne "false") {
     Write-Host "##[info]Build Native simulator"
-    $nativeBuild = (Join-Path $PSScriptRoot "../src/Simulation/Native/build") 
+    $nativeBuild = (Join-Path $PSScriptRoot "../src/Simulation/Native/build")
     cmake --build $nativeBuild --config $Env:BUILD_CONFIGURATION
     if ($LastExitCode -ne 0) {
         Write-Host "##vso[task.logissue type=error;]Failed to build Native simulator."
         $script:all_ok = $False
     }
+
+    Write-Host "##[info]Build QIR Runtime"
+    $oldCC = $env:CC
+    $oldCXX = $env:CC
+    if (-not (Test-Path Env:AGENT_OS) -or ($Env:AGENT_OS.StartsWith("Win"))) {
+        $env:CC = "C:\Program Files\LLVM\bin\clang.exe"
```

Comment:
how does this normally get installed? Should be listed in README.md

Reply:
The prerequisites are currently listed in the readme in QirRuntime folder on the assumption that at the moment it's completely additive and not required for anything else in qsharp-runtime, so someone will opt-in to contribute to QirRuntime. I'll look into providing the controls to exclude it from the local builds by default while allowing us to run CI. Ideally, the CI should trigger only when there are changes to QirRuntime itself and to the native simulator. Is that possible?

Reply:
We will look into facilitating setup later, when QirRuntime becomes required part of the qsharp-runtime. Right now nobody is expected to depend on it.